lib/internal/main/mksnapshot.js

```diff
@@ -134,6 +151,17 @@ function main() {
   } else {
     serializeMainFunction(requireForUserSnapshot, filename, dirname);
   }
+
+  addSerializeCallback(() => {
+    stackTraceLimitDesc = ObjectGetOwnPropertyDescriptor(Error, 'stackTraceLimit');
+
+    if (stackTraceLimitDesc !== undefined) {
+      ObjectSetPrototypeOf(stackTraceLimitDesc, null);
```

Comment:
I realize my original suggestion didn't include a comment, and since you were asking yourself, it's probably a good idea to add one.
```suggestion
      // We want to use null-prototype objects to not rely on globally mutable
      // %Object.prototype%.
      ObjectSetPrototypeOf(stackTraceLimitDesc, null);
```